packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -408,19 +510,44 @@ def imshow(
     else:
         raise ValueError(
             "px.imshow only accepts 2D single-channel, RGB or RGBA images. "
-            "An image of shape %s was provided" % str(img.shape)
+            "An image of shape %s was provided."
+            "Alternatively, 3- or 4-D single or multichannel datasets can be"
+            "visualized using the `facet_col` or/and `animation_frame` arguments."
+            % str(img.shape)
         )
 
-    layout_patch = dict()
+    # Now build figure
+    col_labels = []
+    if facet_col is not None:
+        slice_label = (
+            "slice" if labels.get("facet_col") is None else labels["facet_col"]
```

Comment:
just still need to change `"slice"` to `"facet_col"` here please

Reply:
thanks for your patience ;-)

Reply:
:)